src/test/mod.rs

```diff
@@ -27,17 +27,25 @@ use self::harness::{expect_messages, ExpectedMessage, init_env, mock_server, moc
 
 use url::Url;
 use ls_types::*;
+use lsp_data::InitializationOptions;
 use std::path::Path;
 
 impl ServerMessage {
     pub fn initialize(id: usize, root_path: Option<String>) -> ServerMessage {
+         Self::initialize_with_opts(id, root_path, None)
+    }
+
+    pub fn initialize_with_opts(id: usize, root_path: Option<String>, initialization_options: Option<InitializationOptions>) -> ServerMessage {
```

Comment:
oh of course, sorry, I thought this was still in server.rs, I didn't see it was in the tests